OpenSim/Tools/InverseKinematicsTool.h

```diff
@@ -164,6 +166,11 @@ OpenSim_DECLARE_CONCRETE_OBJECT(InverseKinematicsTool, Tool);
     //--------------------------------------------------------------------------
     bool run() override SWIG_DECLARE_EXCEPTION;
 
+    // For testing/debugging it is necessary to know exactly what are the
+    // MarkersReference (set of marker trajectories and their weights) and
+    // CoordinateReferences that are being used by the InverseKinematicsSolver.
+    void populateReferences(MarkersReference& markersReference,
+        SimTK::Array_<CoordinateReference>&coordinateReferences) const;
 
```

Comment:
Is there a reason for this method to be in the public interface? Could this be kept private?